### ISSUE TITLE
Accessibility Issues In Dark High Contrast Theme When Using Source Control

### ISSUE THREAD

**Issue by d-01** [microsoft/vscode#150595]
Issue Type: <b>Bug</b>

Steps to reproduce:
1. Activate Dark High Contrast theme.
1. Open Explorer panel (Ctrl+Shift+E).
1. Create new workspace in an empty folder.
1. Add a bunch of files (can be empty).
1. `>Git: Initialize Repository`
1. `>Git: Stage All Changes`

Expected result:

1. All labels are clearly visible.
   * Contrast: no less then `4.5` (Level AA, according to [WCAG 2.1](https://www.w3.org/TR/WCAG21/#contrast-minimum))

Actual result:
1. Poor contrast on tab labels, Explorer items and Open Editors items.
   * Text color: `rgb(27, 82, 37)`
   * Background: `rgb(0, 0, 0)`
   * Contrast: `2.28` (according to dev tools and [contrast-ratio.com](<https://contrast-ratio.com/#rgb(27%2C 82%2C 37)-on-(0%2C0%2C0)>) online tool).

Screenshot:

![image](https://user-images.githubusercontent.com/45796739/170816479-5f2a42e3-d604-4ba9-a6c9-525dc238b567.png)

VS Code version: Code 1.67.2 (c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5, 2022-05-17T18:15:52.058Z)
OS version: Windows_NT x64 6.1.7601
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM)2 Quad CPU    Q6600  @ 2.40GHz (4 x 2400)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: unavailable_off<br>opengl: enabled_on<br>rasterization: unavailable_off<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|6.00GB (1.67GB free)|
|Process Argv|--crash-reporter-id a8433a52-a6c8-40eb-8636-eec7af64dfe2|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
pythonvsnew555:30457759
vscscmwlcmt:30465135
cppdebug:30492333
vsclangdc:30486549

```

</details>

**Comment by lszomoru** [comment]
@daviddossett, could you please take a look? Thank you!

**Comment by daviddossett** [comment]
Thanks for the catch, @d-01. Fixed with #150880

<img width="617" alt="CleanShot 2022-05-31 at 10 00 45@2x" src="https://user-images.githubusercontent.com/25163139/171231733-5cc1ae86-dc82-400a-a621-f310085ddcf8.png">
